layouts/_default/provider.html

```diff
@@ -0,0 +1,7 @@
+{{ define "title" }}
+Authentication provider | {{ .Title }}
+{{ end }}
+
+{{ define "main" }}
+{{ partial "article.html" . }}
+{{ end }}
```

Comment:
As far as I can tell in my tests, this default layout isn't being picked up so you can drop it.

Reply:
Removed!